.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,80 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, "unverified"]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened? Also tell us, what did you expect to happen? If applicable, add screenshots to help explain your problem.
+      description: Tell us what you see!
+      placeholder: 
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-did-you-expected
+    attributes:
+      label: What did you expected to happened
+      description: Tell us what you expected!
+      placeholder: 
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a way to reproduce the problem or at list what you did before getting it.
+      placeholder: 
+    validations:
+      required: true
+
+  - type: dropdown
+    id: system-info
+    attributes:
+      label: System Information
+      description: What operating system you are running the extention on?
+      options:
+        - "Windows 10"
+        - "Windows 11"
+        - "MacOS"
+        - "Linux Mint"
+        - "Debian"
+        - "NixOS"
+        - "ArchLinux"
+        - "Other"
+    validations:
+      required: true
+
+    - type: textarea
+    id: browser-info
+    attributes:
+      label: Browser Information
+      description: What browser you are running the extention on?
+      placeholder: 
+    validations:
+      required: true
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output (if you can). This will be automatically formatted into code, so no need for backticks.
+      render: Shell
+
+    - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context
+      description: Add any other context about the problem here.
+      placeholder: 
+    validations:
+      required: true
```

some people don't have much to say
```suggestion
      required: false
```